### ISSUE TITLE
DLC 3.0 training network selects as "best model" one after only 10 epochs (!)

### ISSUE THREAD

**Issue by DanaeNikol** [DeepLabCut/DeepLabCut#2900]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Bug description

Hi there!
Theanks a lot for your work on Deeplabcut! You definitely make one's work on big data a lot better and easier.

I recently switched from DLC 2.1.10 to DLC 3.0 and am facing a quite strange problem. I have created a new network from scratch. I have labeled 900 frames, out of which 764 went into training using pytorch (the rest of the 900 did not have animals labeled due to weird postures, thus they were automatically not included in the training). 
I used the default settings to run a network training as in the picture below (on pytorch): 

![Image](https://github.com/user-attachments/assets/b05f8dd3-3ffd-4637-bd2e-9579a6bfdfd0)

![Image](https://github.com/user-attachments/assets/a73fb9c8-5f33-497c-baad-6bd50cd21277)

Although the training went through until snapshot 200, when trying to evaluate the network I realized that it was evaluated only at the state of epoch 10. In addition, I could already see from the terminal that the test values were not different every 10 epochs (that's when they were recalculated every time).

As a step to continue, I tried analyzing and creating labeled videos. This also did not work as predictions have likelihood close to 0, which I realized after lowering p-cutoff to 0.0. 

Normally, under the same conditions (so same number of frames fed into the network) on DLC 2.1.10 we would still get a decent result that we could work with. However, this is not the case here. Since overfitting is improbable (since basically we obtain no predictions), I am wondering what could be that is so wrong.

In the project I am working on we are currently using DeepLabcut for the pose estimation of animals that are distinguished based on fur patterns. For this purpose we have been using and designing single animal networks where each body part refers to a different fur pattern. We have been using the single animal version because with the multanimal detection was not good based only on the identity.

Do you have any ideas about what might be wrong? It is the first time trying to train a network on DLC 3.0 using Pytorch. Any help would be extremely appreciated!

### Operating System

Ubuntu24

### DeepLabCut version

DLC 3.0

### DeepLabCut mode

single animal

### Device type

GPU GeForce RTX 2080 Ti 

### Relevant log output

```shell

```

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by n-poulsen** [comment]
Hi @DanaeNikol! Could you please share with me the `train.txt` and learning_stats files that were created?

**Comment by DanaeNikol** [comment]
Hi @n-poulsen! 

Here are the learning_stats and train.txt files

[learning_stats.csv](https://github.com/user-attachments/files/18950764/learning_stats.csv)
[train.txt](https://github.com/user-attachments/files/18950765/train.txt)

**Comment by n-poulsen** [comment]
This is further discussed in [this forum post](https://forum.image.sc/t/evaluation-results-displays-different-number-of-epochs/108459). The "best" model is saved after 10 epochs because the loss is flat and the model isn't learning.

This is due to the difficulty of the dataset.
